src/routes/userRoutes.js

```diff
@@ -43,4 +44,13 @@ router
   .route('/:id')
   .delete(authenticateUser, authorizePermissions('admin'), deleteSingleUser);
 
+// Route for rendering the upload page
+router.get('/:userId/uploadProfile', imgurController.renderUploadPage);
```

Comment:
No, we don't have this function 😊, we have uploadProfile in imgurcontroller

Reply:
Are you planning on adding `renderUploadPage` in this PR or in a future one? Currently since this value doesn't exist, it will cause errors that cause the server not to be able to start up. If we will apply that in a future PR we can comment it out here for now.

This is the error I get when I run `npm run dev` and it points to this line:

```
Error: Route.get() requires a callback function but got a [object Undefined]
    at Route.<computed> [as get] (/Users/aasante/dev/practicum2023/backend/node_modules/express/lib/router/route.js:211:15)
    at Function.proto.<computed> [as get] (/Users/aasante/dev/practicum2023/backend/node_modules/express/lib/router/index.js:521:19)
    at Object.<anonymous> (/Users/aasante/dev/practicum2023/backend/src/routes/userRoutes.js:48:8)
    at Module._compile (node:internal/modules/cjs/loader:1103:14)
    at Object.Module._extensions..js (node:internal/modules/cjs/loader:1157:10)
    at Module.load (node:internal/modules/cjs/loader:981:32)
    at Function.Module._load (node:internal/modules/cjs/loader:822:12)
    at Module.require (node:internal/modules/cjs/loader:1005:19)
    at require (node:internal/modules/cjs/helpers:102:18)
    at Object.<anonymous> (/Users/aasante/dev/practicum2023/backend/src/expressServer.js:122:20)
[nodemon] app crashed - waiting for file changes before starting...
```

Reply:
Personally, I think it would make sense to go ahead and create the matching controller function in this PR since you already have the EJS file and everything else ready.

Reply:
Agree, we can do it within this PR , I have added these lines of code, possibly needs correction, could you check please? commit c9734635973e12cf2dc6a76b51630d1700283fed